aten/src/ATen/native/cuda/Embedding.cu

```diff
@@ -180,15 +180,15 @@ __global__ void renorm_kernel(
     } else if (norm_type == 2) {
       v += x * x;
     } else {
-      v += std::pow(x, norm_type);
+      //v += std::pow(x, norm_type);
```

Comment:
@jithunnair-amd I don't think it's a good way to change the behavior of an operator. I'd be in favor of disabling the whole operator instead of changing its behavior just to get things compiled. It would trigger future issue later on. Moreover not comment has been provided on these changes which makes it very hard to fix them later on.